src/Installers/Windows/AspNetCoreModule-Setup/build/settings/common.props

```diff
@@ -16,8 +16,8 @@
     <IisOobWinSdkVersion Condition="'$(IisOobWinSdkVersion)' == ''">10.0.17134.0</IisOobWinSdkVersion>
     <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)' == ''">$(IisOobWinSdkVersion)</WindowsTargetPlatformVersion>
     <CharacterSet>Unicode</CharacterSet>
-    <OutDir>bin\$(Configuration)\$(PlatformShortname)\</OutDir>
-    <IntDir>obj\$(Configuration)\$(PlatformShortname)\</IntDir>
+    <OutDir Condition=" '$(OutDir)' == '' ">bin\$(Configuration)\$(PlatformShortname)\</OutDir>
```

Comment:
@JunTaoLuo @dougbu do you remember why we need this setting here at all? This file is imported very late, and up until this moment `outdir` points to `bin\Release\Installers`. Signing fails without this change because the files to sign must be in `outdir`, and this setting changes where `outdir` points to. My change didn't affect where installers got built to, but I'm wondering if I'm missing something.

Reply:
Maybe @jkotalik knows something about this? Looks like the change was added in https://github.com/dotnet/aspnetcore/commit/8356baf7a612bd21cc92f8fc7098e292274de102 but that commit is huge.